frontend/client/views/settings/index.vue

```diff
@@ -31,8 +31,12 @@
                         <span>{{ convertTime(props.row.lastlogin) }}</span>
                       </td>
                       <td>
-                        <a v-on:click="editUserModal(props.row)"><i class="fa fa-edit" style="color: whitesmoke;"></i></a>
-                        <a v-on:click="deleteUserModal(props.row)" v-if="props.row.username !== session.username"><i class="fa fa-trash" style="color: whitesmoke;"></i></a>
+                        <a v-on:click="editUserModal(props.row)"><i class="fa fa-edit"
+                                                                    style="color: whitesmoke;"></i></a>
+                        <a v-on:click="userPermissionsModal(props.row)"><i class="fa fa-user"
```

Comment:
It seems like this modal doesn't exist anymore?

Reply:
Very good point mate! Thanks for pointing it out